.eslintrc.js

```diff
@@ -19,6 +19,15 @@ module.exports = {
     jest: true,
     jasmine: true,
   },
+  // Add any other files or directories you want to ignore
+  ignorePatterns: [
```

Comment:
we need to delete `.eslintignore` to verify that this section is equivalent

Reply:
@yurishkuro  .eslintignore is deleted.